hack/release/release.sh

```diff
@@ -2,20 +2,15 @@
 set -euo pipefail
 
 GIT_TAG=$(git describe --exact-match --tags || echo "no tag")
+if [[ "$GIT_TAG" == "no tag" ]]; then
+    echo "Failed to release: commit is untagged"
+    exit 1
+fi
 HEAD_HASH=$(git rev-parse HEAD)
 
 SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 source "${SCRIPT_DIR}/common.sh"
-
 config
-release "$HEAD_HASH" #release a snapshot version
 
-## Reset and Clean changes if it's a snapshot since we don't need to track these changes
-## and results in a -dirty commit hash for a stable release
-git reset --hard
-git clean -qdxf
```

Comment:
Did we want to remove this?

Reply:
Yes, we now don't do a snapshot release at the same time that we do a regular release since it was unneccesary. This just pared it down to _just_ do the official release.